### ISSUE TITLE
how about chinese

### ISSUE THREAD

**Issue by 779144625** [WFCD/WFinfo#297]
I am a native Chinese speaker. May I ask if you have any inclination to sinicize? Or need any help?

**Comment by D1firehail** [comment]
I don't believe there's any active work on supporting more languages. Some people have worked on it in the past, but either stopped due to lack of time/motivation or haven't responded to feedback on what their solution is missing before it can be accepted.

For clarity, the prices used by WFInfo correspond to the global version of Warframe. If you're playing the china-specific version and not the global version with language set to chinese, the displayed prices would not be accurate.
